Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -435,6 +435,768 @@ script:
+    description: Retrieves details for the specified asset to include custom attributes.
+    name: tenable-io-export-assets
+    outputs:
+    - contextPath: TenableIO.Asset.id
+      description: The UUID of the asset in Tenable.io. Use this value as the unique key for the asset
+      type: String
+    - contextPath: TenableIO.Asset.has_agent
+      description: Specifies whether a Nessus agent scan identified the asset.
+      type: Boolean
+    - contextPath: TenableIO.Asset.has_plugin_results
+      description: Specifies whether the asset has plugin results associated with it.
+      type: Boolean
+    - contextPath: TenableIO.Asset.created_at
+      description: The time and date when Tenable.io created the asset record.
+      type: Date
+    - contextPath: TenableIO.Asset.terminated_at
+      description: The time and date when a user terminated the Amazon Web Service (AWS) virtual machine instance of the asset.
+      type: Date
+    - contextPath: TenableIO.Asset.terminated_by
+      description: The user who terminated the AWS instance of the asset.
+      type: String
+    - contextPath: TenableIO.Asset.updated_at
+      description: The time and date when the asset record was last updated.
+      type: Date
+    - contextPath: TenableIO.Asset.deleted_at
+      description: The time and date when a user deleted the asset record. When a user deletes an asset record, Tenable.io retains the record until the asset ages out of the license count.
+      type: Date
+    - contextPath: TenableIO.Asset.deleted_by
+      description: The user who deleted the asset record.
+      type: String
+    - contextPath: TenableIO.Asset.first_seen
+      description: The time and date when a scan first identified the asset.
+      type: Date
+    - contextPath: TenableIO.Asset.last_seen
+      description: The time and date of the scan that most recently identified the asset.
+      type: Date
+    - contextPath: TenableIO.Asset.first_scan_time
+      description: The time and date of the first scan run against the asset.
+      type: Date
+    - contextPath: TenableIO.Asset.last_scan_time
+      description: The time and date of the last scan run against the asset.
+      type: Date
+    - contextPath: TenableIO.Asset.last_authenticated_scan_date
+      description: The time and date of the last credentialed scan run on the asset.
+      type: Date
+    - contextPath: TenableIO.Asset.last_licensed_scan_date
+      description: The time and date of the last scan that identified the asset as licensed. Tenable.io categorizes an asset as licensed if a scan of that asset has returned results from a non-discovery plugin within the last 90 days.
+      type: Date
+    - contextPath: TenableIO.Asset.last_scan_id
+      description: The UUID of the scan configuration used during the last scan of the asset.
+      type: String
+    - contextPath: TenableIO.Asset.last_schedule_id
+      description: The schedule_uuid for the last scan of the asset.
```

Comment:
Should the description be 
- The schedule_**id** for the last scan of the asset.
or should the contextPath be:
- TenableIO.Asset.last_schedule_**uuid**